inst/compareScenarios/cs_02_macro.Rmd

```diff
@@ -203,15 +203,53 @@ vars <- intersect(paste0("Price|Final Energy|Transport|", carriers, "|Rawdata"),
 walk(vars, showLinePlots, data = dPrice, scale = "fixed")
 ```
 
+### SE Imports Prices
+
+```{r SE Imports Prices}
+vars <-
+  levels(data$variable) %>%
+  str_subset("^Trade\\|Price\\|Imports\\|SE\\|")
+walk(vars, showLinePlots, data = data)
+```
+
+### SE Exports Trade
+
+```{r SE Exports Trade}
+vars <-
+  levels(data$variable) %>%
+  str_subset("^Trade\\|Price\\|Exports\\|SE\\|")
+walk(vars, showLinePlots, data = data)
+```
+
 ## Trade
 
-```{r Trade}
+### PE Trade
+
+```{r PE Trade}
 vars <-
   levels(data$variable) %>%
   str_subset("^Trade\\|(Uranium.)?[^|]*$")
 walk(vars, showLinePlots, data = data)
 ```
 
+### SE Imports Trade
```

Comment:
was there a specific reason you decided to plot imports and exports separately? 
for PE, we only show the net trade, and I think that is sufficient information also for SE in the compScen, as there should be no region that both imports and exports SE